### PR TITLE
[12.0] partner noncommercial streamline

### PR DESCRIPTION
**Pull request by NL66278**
- Remove window action / menu to show all contacts, as it duplicates the main Contacts action;
- Make sure general contact configuration menu is shown after added menu's;
- Move some selections from context to domain, to ensure view always reflects its purpose;
- Add some filters / groupings nearly always needed in non commercial settings (but usefull in  general).

### CODE REVIEW

**Review by lfreeke — APPROVED** [review]
LGTM Functional review

**Comment by NL66278** [comment]
Updated this after comments from customer, clearly showing there is no need from a separate menu, apart from the main contacts menu-option. The extra filters and non-commercial terminology are satisfactory.

File: partner_noncommercial/readme/USAGE.rst [hunk 1]
```diff
@@ -1,3 +1,3 @@
-This module creates a main menu All partners. There it gives special
+This module creates a main menu Partners. There it gives special
```
Comment (thomaspaulb) [inline partner_noncommercial/readme/USAGE.rst]: This is technically not true any more, we should maybe change this to:

"This module adds specific filters, groupings and modifies some naming to make the existing main menu Contacts easier to use for non-commercial entities".

File: partner_noncommercial/views/res_partner.xml [hunk 2]
```diff
@@ -4,24 +4,52 @@
...
+                    string="City"
+                    context="{'group_by': 'city'}"
+                    />
+                <filter
+                    name="group_organisation"
+                    string="Organisation"
+                    context="{'group_by': 'commercial_partner_id'}"
+                    />
+            </filter>
+
```
Comment (thomaspaulb) [inline partner_noncommercial/views/res_partner.xml:52]: Can you do an interactive rebase to get the whole bunch of commits down to two: adding of the filters/grouping, and removal of the menus?

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

File: partner_noncommercial/readme/USAGE.rst (hunk 1 above)
Comment (NL66278) [reply partner_noncommercial/readme/USAGE.rst]: I adapted the usage.

**Review by thomaspaulb — APPROVED** [review]

**Pull request merged**